content/en/agent/guide/install-agent-6.md

```diff
@@ -0,0 +1,1240 @@
+1. Restart your Puppet server to begin receiving Puppet data in Datadog.
+
+## Specific Agent checks
+
+To use a specific Agent check or integration on one of your nodes, see the relevant [integration manifest][102] for a code sample. Here is an example for the elasticsearch integration:
+
+```puppet
+node "elastic-node1.mydomain.com" {
+    class { "datadog_agent":
+        api_key => "MY_API_KEY",
+        datadog_site => "datad0g.com",
+        agent_major_version => 6,
+    }
+    include "datadog_agent::integrations::elasticsearch"
+}
+```
+
+Refer to the [GitHub repository of the module][103] for more examples and advanced use cases.
+
+[101]: https://forge.puppetlabs.com/modules/datadog/datadog_agent/readme
+[102]: https://github.com/DataDog/puppet-datadog-agent/tree/main/manifests/integrations
+[103]: https://github.com/DataDog/puppet-datadog-agent
+[104]: 
+
+{{% /tab %}}
+
+{{% tab "Chef" %}}
+
+<div class="alert alert-info">Requires Chef version 10.14.x or higher.</a></div>
+
+1. Add the Datadog cookbook:
+   - If you are using [Berkshelf][101], add the cookbook to your Berksfile:
+      ```shell
+      cookbook 'datadog', '~> 4.0'
+      ```
+
+   - If you're not using Berkshelf, install the cookbook in to your repository using Knife:
+     ```shell
+     knife cookbook site install datadog 
+     ```
+
+1. Set the Datadog-specific attributes in either a role, environment, or another recipe. Replace `MY_API_KEY` with your Datadog API key:
+   ```chef
+   node.default['datadog']['api_key'] = "MY_API_KEY"
+
+   # Use an existing application key or create a new one for Chef
+   node.default['datadog']['application_key'] = "Generate Application Key"
+
+   # Enable install for Agent version 6
+   node.default['datadog']['agent_major_version'] = 6
+
+   # Set the Datadog site
+   node.default['datadog']['site'] = "datad0g.com"
+   ```
+
+1. Upload the updated cookbook to your Chef server:
+   ```shell
+   berks upload
+   # or
+   knife cookbook upload datadog
+   knife cookbook list | grep datadog && 
+   echo -e "e[0;32mdatadog cookbook - OKe[0m" ||
+   echo -e "e[0;31mmissing datadog cookbook - OKe[0m"
+   ```
+
+1. Add the cookbook to your node's `run_list` or `role`:
+   ```chef
+   "run_list": [
+    "recipe[datadog::dd-agent]"
+   ]
+   ```
+
+1. Wait for the next scheduled `chef-client` run.
+
+
+For more information and examples, see the [Agent GitHub repository][102].
+
+[101]: https://docs.chef.io/workstation/berkshelf/
+[102]: https://github.com/DataDog/chef-datadog
+
+{{% /tab %}}
+
+{{% tab "SaltStack" %}}
+
+<div class="alert alert-info">The Datadog Saltstack formula only supports Debian-based and RedHat-based systems.<br><br>
+The following instructions add the Datadog formula to the <code>base</code> Salt environment. To add it to another Salt environment, replace references to <code>base</code> with the name of your Salt environment.</div>
+
+<!-- vale Datadog.inclusive = NO -->
```

Comment:
Is this supposed to be left here?

Reply:
Yes, it prevents Vale from flagging "master"